vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -1,15 +1,10 @@
 <template>
     <div class="form-input" v-bind:class="classes">
         <slot name="header">
-            <label-ripe
-                class="header"
-                v-bind:style="headerStyle"
-                v-bind:class="headerClasses"
-                v-bind:size="headerSize"
-                v-bind:text="header"
-                v-bind:for="id"
-                v-if="header"
-            />
+            <tooltip v-bind="tooltipProps" v-if="hasTooltip">
+                <label-ripe class="header" v-bind="headerProps" v-if="header" />
+            </tooltip>
+            <label-ripe class="header" v-bind="headerProps" v-else />

```

Comment:
This is changing the header logic, it should be `v-else-if="header"` instead of `v-else`